.github/workflows/nodejs.yml

```
@@ -10,6 +10,67 @@ on:
+          # Opens tcp port 6379 on the host and service container
+          - 6379:6379
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [18.20.0, 18, 20, 22]
+        os: [ubuntu-latest]
+
+    steps:
+    - name: Checkout Git Source
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)